### PR TITLE
feat(builtin): builtinComponent 组件独立包抽离

### LINKED ISSUE #19: 🐛 [Bug]: CanvasRowColContainer.vue组件 拖拽后预览时报找不到标签
### Environment

nodejs v18.18.0 

### Version

latest

### Version

latest

### Link to minimal reproduction

about:srcdoc:160 [Vue warn]: Failed to resolve component: canvas-col
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <Main> 
  at <Repl>
console.<computed> @ about:srcdoc:160
about:srcdoc:160 [Vue warn]: Failed to resolve component: canvas-row
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <Main> 
  at <Repl>
console.<computed> @ about:srcdoc:160
about:srcdoc:160 [Vue warn]: Failed to resolve component: canvas-row-col-container
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <Main> 
  at <Repl>

### Step to reproduce

CanvasRowColContainer.vue component cannot find the label when previewing after dragging

### What is expected

_No response_

### What is actually happening

_No response_

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by chilingling**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

**Comment by chilingling** [comment]
also fix #19

File: packages/builtinComponent/src/meta/CanvasCol.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
+{
+    "components": {
+        "icon": "Box",
```
Comment (wenmine) [inline packages/builtinComponent/src/meta/CanvasCol.json]: 这个文件代码格式和其他json文件不同，请保持一致

Comment (chilingling) [reply packages/builtinComponent/src/meta/CanvasCol.json]: Done

**Review by wenmine — DISMISSED** [review]
LGTM

**Review by ghost — DISMISSED** [review]
LGTM

File: packages/builtinComponent/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# 内置组件
+
+## 目前内置的组件
+
+### CanvasRow
+
+### CanvasCol
+
+### CanvasRowContainer
```
Comment (ghost) [inline packages/builtinComponent/README.md]: CanvasRowColContainer

Comment (chilingling) [reply packages/builtinComponent/README.md]: Done

**Review by ghost — APPROVED** [review]

**Review by wenmine — APPROVED** [review]

**Pull request merged**